x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_disabled_fields_header.test.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { ResultSettingsDisabledFieldsHeader } from './result_settings_disabled_fields_header';
+
+describe('ResultSettingsDisabledFieldsHeader', () => {
+  it('renders', () => {
+    const wrapper = shallow(<ResultSettingsDisabledFieldsHeader />);
+    expect(wrapper.isEmptyRender()).toBe(false);
```

Comment:
Kind of a lame test, but there wasn't really anything to test, it's all pretty static. Let me know if you want me to improve this.

Reply:
Nah, `isEmptyRender()` for super small & static components is fine IMO.

Alternatively if you want to try experimenting with `jest.snapshot()` feel free, but I'm good wit this!